### PR TITLE
Added interregional electricity flows

### PR DESCRIPTION
**Pull request by amosschle**
Interregional electricity flows are required to upload CS7 results (Danish electricity imports from and exports to neighbour countries). As far as I understand, the current `Import|Electricity` and `Export|Electricity` do not allow specifying two regions.

### CODE REVIEW

File: definitions/variable/technology/electricity-operation.yaml [hunk 1]
```diff
@@ -108,6 +108,9 @@
 - Import|Electricity:
     description: Exported electricity from a given region
     unit: MWh
+- Flow|Electricity|{Region}|{Region}:
```
Comment (sandrinecharousset) [inline definitions/variable/technology/electricity-operation.yaml:111]:
```suggestion
- Flow|Electricity:
```

Comment (sandrinecharousset) [reply definitions/variable/technology/electricity-operation.yaml:111]: the column region is meant for describing the region this variable applies to. Directionnal regions are already included and have to be written region1 > region2 (in the region column), which here means that the energy flow is from region1 to region2 (can be negative which means the flow is from region2 to region1)

**Review by sandrinecharousset — CHANGES_REQUESTED** [review]

File: definitions/variable/technology/electricity-operation.yaml (hunk 1 above)
Comment (amosschle) [reply definitions/variable/technology/electricity-operation.yaml:111]: Thanks for the quick review and the info, I didn't know about directional regions. I'll commit your suggestion.

File: definitions/variable/technology/electricity-operation.yaml [hunk 2]
```diff
@@ -108,6 +108,9 @@
 - Import|Electricity:
     description: Exported electricity from a given region
     unit: MWh
+- Flow|Electricity:
+    description: Flow from electricity from the former to latter region
+    unit: MWh
```
Comment (sandrinecharousset) [inline definitions/variable/technology/electricity-operation.yaml:113]:
```suggestion
    unit: [MWh, GWh]
```

**Review by sandrinecharousset — CHANGES_REQUESTED** [review]
propose to allow both MWh and GWh as unit, as this variable will represent an 'energy' flow, and energies are often expressed in GWh

**Review by danielhuppmann — APPROVED** [review]
Thanks @amosschle

**Comment by danielhuppmann** [comment]
Tests reminded me that this variable was already added by #280, see [here](https://github.com/openENTRANCE/openentrance/blame/main/definitions/variable/technology/electricity-operation.yaml#L120)

**Pull request closed without merging**